src/Symfony/Bridge/Twig/Extension/CodeExtension.php

```diff
@@ -137,7 +137,7 @@ public function fileExcerpt(string $file, int $line, int $srcContext = 3): ?stri
             }
 
             for ($i = max($line - $srcContext, 1), $max = min($line + $srcContext, \count($content)); $i <= $max; ++$i) {
-                $lines[] = '<li'.($i == $line ? ' class="selected"' : '').'><a class="anchor" name="line'.$i.'"></a><code>'.self::fixCodeMarkup($content[$i - 1]).'</code></li>';
+                $lines[] = '<li'.($i == $line ? ' class="selected"' : '').'><a class="anchor" id="line'.$i.'"></a><code>'.self::fixCodeMarkup($content[$i - 1]).'</code></li>';
```

Comment:
to avoid duplicate IDs we could use a static uniquely prefixed counter perhaps

Reply:
well no, because we dont know the ID upfront for linking. Not sure what's best :P

Reply:
another "oh well", currently multiple code blocks on the same probably cant be linked to already.

But this only introduces more errors then.

Perhaps keep the name attr./deprecation in this case, given the HTML is exposed in public domain.

Reply:
`filename|file_excerpt(line, -1, line_anchors=true)` explicitly on the 'profiler open page' will also do it, then using id attribute is ok.

Reply:
This code change is only for good measures honestly, it doesn't change anything about the behavior.
Duplicate `name`attributes are equally as bad as duplicate `id` attributes, not better nor worse.

Also, I don't think this is a big issue as `CodeExtension` is used for viewing single files.
Anyways, this is a pre-existing problem and is outside the scope of this PR.

Reply:
>  not better nor worse

works for me :+1: